content/extending-ui/extending-chat-widget/rich-messages/index.mdx

```diff
@@ -0,0 +1,267 @@
+
+## Showing off your products
+If you are running an online shop, there must be dozens of situations where Customers ask for your advice. With the use of Rich Messages, you can present your goods with links way more as a more attractive Carousel:
+
+<CodeSample>
+
+```json
+{
+	"template_id": "cards",
+	"elements": [{
+			"title": "Tomatoes",
+			"subtitle": "The tomato is the edible, often red, berry of the plant Solanum lycopersicum, commonly known as a tomato plant.",
+			"image": {
+				"url": "https://images.pexels.com/photos/1327838/pexels-photo-1327838.jpeg"
+			},
+			"buttons": [{
+				"type": "url",
+				"text": "Open link",
+				"postback_id": "open_url",
+				"user_ids": [],
+				"value": "https://example.org/?product1"
+			}]
+		},
+		{
+			"title": "Cucumber",
+			"subtitle": "There are three main varieties of cucumber: slicing, pickling, and seedless. Within these varieties, several cultivars have been created.",
+			"image": {
+				"url": "https://images.pexels.com/photos/37528/cucumber-salad-food-healthy-37528.jpeg"
+			},
+			"buttons": [{
+				"type": "url",
+				"text": "Open link",
+				"postback_id": "open_url",
+				"user_ids": [],
+				"value": "https://example.org/?product2"
+			}]
+		}
+	]
+}
+```
+
+</CodeSample>
+
+Your Customer would see the following in the Widget:
+
+![Products Example Placeholder](images/products-example-placeholder.png)
+
+This way, they can visit product pages and review them much easier!
+
+## Sharing knowledge with your Customer
+[Moments](/extending-ui/extending-chat-widget/chat-widget-moments/) are powerful Apps, yet the `webview` can also contain static websites. Let's assume, once again, that you run Support team and would like to share an article explaining simple steps to undertake:
+
+<CodeSample>
+
+```json
+{
+	"template_id": "cards",
+	"elements": [{
+		"title": "How to?",
+		"subtitle": "Let me guide you through the simple steps you should undertake.",
+		"buttons": [{
+			"type": "webview",
+			"text": "Open link",
+			"postback_id": "open_url",
+			"user_ids": [],
+			"value": "https://example.org/?how-to",
+			"webview_height": "full"
+		}]
+	}]
+}
+```
+
+</CodeSample>
+
+That would show a simple Card with a button, which after being clicked would open the article right in the Chat:
+
+![Webview Example Placeholder](images/webview-example-placeholder.png)
+
+# How to use Rich Messages in LiveChat?
+Currently there are several ways to use Rich Messages in the LiveChat.
+
+## ChatBot
+Rich Messages are very important feature of [ChatBot](https://www.livechatinc.com/chatbot/). They allow the Bot to connect easier with your Customers, making sure that they follow the interactions you thought of. With the [Rich Messages](https://www.livechatinc.com/help/rich-messages/) and easy configurator, Bot can make your Customer's chat a real pleasure.
+
+![Chatbot integration Placeholder](images/chatbot-integration-placeholder.png)
+
+## Product Cards for BigCommerce
```

Comment:
We have Product Cards for Shopify, too.
We have also "white label" Product Cards app - https://developers.livechatinc.com/docs/extending-ui/extending-agent-app/product-cards/#header - developer can prepare own backend and connect ready to use User interface.

We can also give other samples of that make use of rich messages:
- Calendly
- join.me
- KnowledgeBase